src/components/HomePage/HomePage.css

```diff
@@ -39,6 +39,20 @@
   position: relative;
 }
 
+.HomePage .empty-projects {
+  display: flex;
+  width: 100%;
+  height: 200px;
+  border-radius: 12px;
+  border: 1px solid var(--bluish-steel);
```

Comment:
```suggestion
  border: 1px solid var(--divider);
```

Reply:
I would use the `divider` variable, that is meant for those kind of lines, it's a little darker:

<img width="1438" alt="Screen Shot 2019-08-16 at 3 23 59 PM" src="https://user-images.githubusercontent.com/2781777/63189241-e6d68c80-c039-11e9-881f-62374ebc4c9a.png">